dev/README_RELEASE_AIRFLOW.md

```diff
@@ -805,10 +805,7 @@ Documentation for providers can be found in the ``/docs/apache-airflow`` directo
 
     ```shell script
     breeze release-management publish-docs --package-filter apache-airflow --package-filter docker-stack
-    cd "${AIRFLOW_SITE_DIRECTORY}"
-    cd post-docs
-    python add-back-references.py airflow
-    cd ..
+    breeze release-management add-back-references --airflow-site-director DIRECTORY --gen-type airflow
```

Comment:
You should not add the flag here. Those are real copy-pasteable instructions, it should **just-work** 

You should remove the flags, and ENVVAR will kick in here. If you look up AIRLFOW_SITE_DIRECTORY is set few steps above , so just running it without the flag should work here (and in the other places).

Reply:
Okay makes sense. Corrected it!

Reply:
What I mean is to remove the flag altogether - this is how it works